internal/db/schema/migrations/oss/postgres/35/01_job_migrations.up.sql

```diff
@@ -0,0 +1,18 @@
+begin;
+
+-- The job_run table has a foreign key to the server_controller private_id.
+-- While this column is called a prviate_id it does not implement the wt_private_id
+-- domain type, and therefore does not have a minimum length of 10 characters.
+
+-- Drop constraint that requires controller_id to be at least 10 chars
```

Comment:
can you add a comment here that references where this constraint was originally added? And can you update that migration to add a comment noting that it was removed in this migration?